core/scripts/chaincli/handler/report.go

```diff
@@ -322,69 +319,69 @@ func (t *OCR2TransmitTx) UpkeepsInTransmit() ([]plugintypes.UpkeepResult, error)
 	return t.encoder.DecodeReport(reportBytes)
 }
 
-func (t *OCR2TransmitTx) SetStaticValues(elem *OCR2ReportDataElem) {
-	if t.To() != nil {
-		elem.To = t.To().String()
-	}
-
-	elem.ChainID = t.ChainId().String()
-
-	from, err := t.From()
-	if err != nil {
-		elem.Err = err.Error()
-		return
-	} else {
-		elem.From = from.String()
-	}
-
-	block, err := t.BlockNumber()
-	if err != nil {
-		elem.Err = err.Error()
-		return
-	} else {
-		elem.BlockNumber = fmt.Sprintf("%d", block)
-	}
-
-	upkeeps, err := t.UpkeepsInTransmit()
-	if err != nil {
-		elem.Err = err.Error()
-	}
-
-	keys := []string{}
-	chkBlocks := []string{}
-	for _, u := range upkeeps {
-		parts := strings.Split(u.Key.String(), "|")
-		keys = append(keys, parts[1])
-		chkBlocks = append(chkBlocks, fmt.Sprintf("%d", u.CheckBlockNumber))
-	}
-	elem.PerformKeys = strings.Join(keys, "\n")
-	elem.PerformBlockChecks = strings.Join(chkBlocks, "\n")
-}
-
-func (t *OCR2TransmitTx) BatchElem() (rpc.BatchElem, error) {
-
-	bn, err := t.BlockNumber()
-	if err != nil {
-		return rpc.BatchElem{}, err
-	}
-
-	from, err := t.From()
-	if err != nil {
-		return rpc.BatchElem{}, err
-	}
-
-	return rpc.BatchElem{
-		Method: "eth_call",
-		Args: []interface{}{
-			map[string]interface{}{
-				"from": from.Hex(),
-				"to":   t.To().Hex(),
-				"data": hexutil.Bytes(t.tx.Data()),
-			},
-			hexutil.EncodeBig(big.NewInt(int64(bn) - 1)),
-		},
-	}, nil
-}
+//func (t *OCR2TransmitTx) SetStaticValues(elem *OCR2ReportDataElem) {
+//	if t.To() != nil {
+//		elem.To = t.To().String()
+//	}
+//
+//	elem.ChainID = t.ChainId().String()
+//
+//	from, err := t.From()
+//	if err != nil {
+//		elem.Err = err.Error()
+//		return
+//	} else {
+//		elem.From = from.String()
+//	}
+//
+//	block, err := t.BlockNumber()
+//	if err != nil {
+//		elem.Err = err.Error()
+//		return
+//	} else {
+//		elem.BlockNumber = fmt.Sprintf("%d", block)
+//	}
+//
+//	upkeeps, err := t.UpkeepsInTransmit()
+//	if err != nil {
+//		elem.Err = err.Error()
+//	}
+//
+//	keys := []string{}
+//	chkBlocks := []string{}
+//	for _, u := range upkeeps {
+//		parts := strings.Split(u.Key.String(), "|")
+//		keys = append(keys, parts[1])
+//		chkBlocks = append(chkBlocks, fmt.Sprintf("%d", u.CheckBlockNumber))
+//	}
+//	elem.PerformKeys = strings.Join(keys, "\n")
+//	elem.PerformBlockChecks = strings.Join(chkBlocks, "\n")
+//}
+
+//func (t *OCR2TransmitTx) BatchElem() (rpc.BatchElem, error) {
+//
+//	bn, err := t.BlockNumber()
+//	if err != nil {
+//		return rpc.BatchElem{}, err
+//	}
+//
+//	from, err := t.From()
+//	if err != nil {
+//		return rpc.BatchElem{}, err
+//	}
+//
+//	return rpc.BatchElem{
+//		Method: "eth_call",
+//		Args: []interface{}{
+//			map[string]interface{}{
+//				"from": from.Hex(),
+//				"to":   t.To().Hex(),
+//				"data": hexutil.Bytes(t.tx.Data()),
+//			},
+//			hexutil.EncodeBig(big.NewInt(int64(bn) - 1)),
+//		},
+//	}, nil
+//}
```

Comment:
do we need these functions? if not let's :scissors: them

Reply:
fixed